src/certifier_tests.mak

```diff
@@ -89,20 +104,27 @@ channel_dobj += $(sev_common_objs)
 pipe_read_dobj += $(sev_common_objs)
 endif
 
-all:	certifier_tests.exe test_channel.exe pipe_read_test.exe
+# RESOLVE: Same issue with ordering and use of CFLAGS_PIC = -fpic for all objects.
```

Comment:
Note to myself: Fix this RESOLVE issue ...

Reply:
Resolved. Added new target `sharedlib` and make command will need to be invoked using `-B` or `--always-make` argument when building this target.

Updated commands in `build.yml` to do a 2nd build to build the shared libraries.